### PR TITLE
feat(ui-ux): add initial smart contract integration for EVM to DFC transfer

### PR DESCRIPTION
**Pull request by lykalabrada**
#### What this PR does / why we need it:
Implement smart contract's `bridgeToDeFiChain` to get funds from EVM using Goerli testnet.

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/ConfirmTransferModal.tsx [hunk 1]
```diff
@@ -118,22 +136,111 @@ function RowData({
...
+      {isSuccess && (
+        // TODO: Replace success ui/message
+        <Modal isOpen={isSuccess} onClose={() => router.reload()}>
+          <div className="flex flex-col items-center mt-6 mb-14">
+            <FiCheck className="text-8xl text-valid ml-1" />
+            <span className="font-bold text-2xl text-dark-900 mt-12">
+              Transaction confirmed
+            </span>
+            <span className="text-dark-900 mt-2">
+              Funds will be transferred to your DefiChain wallet shortly.
```
Comment (thedoublejay) [inline apps/web/src/components/ConfirmTransferModal.tsx:206]:
```suggestion
              Funds will be transferred to your DeFiChain wallet shortly.
```

File: apps/web/src/components/ConfirmTransferModal.tsx [hunk 2]
```diff
@@ -186,96 +292,67 @@ export default function ConfirmTransferModal({
...
+              position="right"
             />
           </div>
+        </div>
+        <NumericFormat
+          className="text-right text-dark-900 tracking-[0.01em] md:tracking-normal"
+          value={0}
+          decimalScale={2}
+          thousandSeparator
+          suffix=" DFI" // TODO: Create hook to get fee based on source/destination
```
Comment (thedoublejay) [inline apps/web/src/components/ConfirmTransferModal.tsx:322]: We have something similar on LW, I think we can use that

File: apps/web/src/components/ConfirmTransferModal.tsx [hunk 3]
```diff
@@ -186,96 +292,67 @@ export default function ConfirmTransferModal({
...
-
-          {isSendingToDFC ? (
-            <ERC20ToDeFiChainTransfer />
-          ) : (
-            <DeFiChainToERC20Transfer />
-          )}
         </div>
-      </Dialog.Panel>
-    </Dialog>
+        <div>
```
Comment (thedoublejay) [inline apps/web/src/components/ConfirmTransferModal.tsx:338]: Can you add a TODO here to link this to Scan once available?

File: apps/web/src/layouts/contexts/ContractContext.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+export function ContractProvider({
+  children,
+}: PropsWithChildren<{}>): JSX.Element | null {
+  const { networkEnv } = useNetworkEnvironmentContext();
+  const [config, setConfig] = useState(MAINNET_CONFIG);
+
+  useEffect(() => {
+    if (networkEnv === NetworkEnvironment.mainnet) {
```
Comment (thedoublejay) [inline apps/web/src/layouts/contexts/ContractContext.tsx:25]: We can simplify it to a variable, so we just call setConfig once

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: apps/web/src/components/ConfirmTransferModal.tsx (hunk 2 above)
Comment (lykalabrada) [reply apps/web/src/components/ConfirmTransferModal.tsx:323]: I have added, followed the one we have on LW

File: apps/web/cypress/e2e/bridge.spec.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
+describe.only("Test User Login", () => {
```
Comment (thedoublejay) [inline apps/web/cypress/e2e/bridge.spec.ts]:
```suggestion
describe("Test User Login", () => {
```

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**